### DISCUSSION TITLE
Updating the knowledge base without redeploying the chain

### DISCUSSION THREAD

**Discussion by sidharth-14** [langchain-ai/langserve#536] [Q&A]
Can I update the file which is going as input to chain using loaders, embeddings and vectorestore without redeploying the chain using langserve

**Comment by eyurtsev** [comment]
Yes there are ways to do it. But the solution is more involved. If your app is only serving the RAG use case, then deployment is actually the best solution in this case!

Deployment won't be acceptable if you have a larger app that's used for other purposes as well not just the RAG use case
